loadtest/scenarios/multiple_EOA.js

```diff
@@ -3,21 +3,46 @@ import exec from 'k6/execution';
 import { fundTestAccounts } from '../helpers/init.js';
 import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
 
-let duration = __ENV.LOADTEST_DURATION;
+let setupTimeOut = __ENV.SETUP_TIMEOUT;
```

Comment:
Nitpick: rename to `setupTimeout` (just rename all the occurrences)?
```suggestion
let setupTimeout = __ENV.SETUP_TIMEOUT;
```